{
  try {
    test(leftOperand,rule,rightOperand);
  }
 catch (  AssertionError e) {
    assumeNoException(e);
  }
}
